packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -143,18 +143,19 @@ export const FileSnapshotWriterClassFactory = <TBase extends ReaderConstructor>(
             return super.readBlob(sha);
         }
 
-        public async getVersions(versionId: string, count: number): Promise<api.IVersion[]> {
+        public async getVersions(versionId: string | null, count: number): Promise<api.IVersion[]> {
             // If we already saved document, that means we are getting here because of snapshot generation.
             // Not returning tree ensures that ContainerRuntime.snapshot() would regenerate subtrees for
             // each unchanged data store.
             // If we want to change that, we would need to capture docId on first call and return this.latestWriterTree
             // when latest is requested.
+            if (this.latestWriterTree && (this.docId === versionId || versionId === undefined) || versionId === null) {
```

Comment:
I think this needs to be 

> if (this.latestWriterTree && (this.docId === versionId || versionId === null)) {

I.e. null should be handled same was as docId. And I believe we dropped undefined (which mean no snapshot) long time ago (worth double checking, but clearly API shape does not allow that)